src/tests/ftest/container/delete.py

```diff
@@ -102,13 +105,24 @@ def test_container_delete(self):
                 cont_uuid = uuid.uuid4()
 
             self.container.destroy(force=force, poh=poh, con_uuid=cont_uuid)
+
+            # close container handle, release a reference on pool in client lib
+            # Otherwise test will ERROR in tearDown (pool disconnect -DER_BUSY)
+            if opened:
+                self.container.close()
+
             self.container = None
 
             if expected_result in ['FAIL']:
                 self.fail("Test was expected to fail but it passed.\n")
 
         except DaosApiError as excep:
-            self.d_log.error(excep)
-            self.d_log.error(traceback.format_exc())
+            print(excep, traceback.format_exc())
             if expected_result == 'PASS':
                 self.fail("Test was expected to pass but it failed.\n")
+
+            if opened:
+                self.container.close()
```

Comment:
This will not be executed if line 122 is called.  Is that the intention?  The comment in lines 109-110 seem to indicate otherwise.